apps/betterangels-backend/shelters/enums.py

```diff
@@ -41,9 +42,11 @@ class ShelterChoices(models.TextChoices):
 # Sleeping Details
 @strawberry.enum
 class RoomStyleChoices(models.TextChoices):
-    CONGREGANT = "congregant", _("Congregant (Open)")
```

Comment:
This is going to require a data migration.

You'll likely need to do it in RAW sql.

For old migrations take the enum and put the enum class directly in the migration and update the references

Reply:
take a look at `apps/betterangels-backend/shelters/migrations/0005_spa_alter_generalservice_name_alter_population_name_and_more.py` on how to do a raw sql migration to update the existing data to get around the validation checks.

Reply:
We're still going to have to do a data migration here.